video-library/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgresSecret
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
+volumes:
+  db:
+    driver: local
```

Comment:
not sure jestli tohle potřebuješ... za mě zbytečný

určitě ale musíš doplnit 
networks a nastavit to na bridge